bundles/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/handler/TcpBridgeConfiguration.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.serial.internal.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * Class describing the serial bridge user configuration
+ *
+ * @author Roland Tapken - Initial contribution
+ */
+@NonNullByDefault
+public class TcpBridgeConfiguration extends CommonBridgeConfiguration {
+    /**
+     * IP address or hostname
+     */
+    public @Nullable String address;
+
+    /**
+     * TCP Port
+     */
+    public int port = 0;
+
+    /**
+     * Socket timeout in seconds
+     */
+    public int timeout = 0;
+
+    /**
+     * Keep Alive
+     */
+    public boolean keepAlive = false;
+
+    /**
+     * Reconnection Interval in seconds
+     */
+    public int reconnectInterval = 10;
```

Comment:
Please document default value in readme.md

Reply:
New description:

> Interval in seconds for automatic reconnect after connection failure (defaults to 10 seconds)